src/components/marker.js

```diff
@@ -61,9 +61,10 @@ export default class Marker extends Component {
     };
 
     return createElement('div', {
-      className: 'mapboxgl-marker',
+      className: className,
```

Comment:
I agree allowing custom class name would be desirable for all our control components. Because we rely on Mapbox's stylesheet, some of the native class names are still necessary. Could you change this to `mapboxgl-marker ${className}` (and maybe add the prop to `Popup` and `NavigationControl` as well)?

Reply:
Yes, I can work on it